.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Comment:
Not that we probably would, but what happens if we get 12 updates? Do the last two get queued up for "by hand" submission?  Or am I misinterpreting this?  If I'm on, I think I'd feel more comfortable at 15 or 20.

Reply:
That should be scheduled for the next day. @saschagrunert, did you set 10 for a specific reason or was it a gut feeling?

Reply:
Thanks @saschagrunert .  So if it doesn't create number 11 and 12, is a notification sent out or will it just do those the next day?